Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Numerics;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Vector2 = UnityEngine.Vector2;
+using Vector3 = UnityEngine.Vector3;
+
+public class PlayerController : MonoBehaviour
+{
+    private EmPControls _controls;
+    private CharacterController _characterController;
+    public Transform cameraTransform;
+    public float gravity = -9.81f;
+    private float _yVelocity;
+    public float terminalVelocity = -10f;
+    public float jumpForce = 5f;
+    public float mouseSensitivity = 0.8f;
+    // Start is called before the first frame update
+    void Start()
+    {
+        _controls = new EmPControls();
+        _controls.Enable();
+        _controls.Ingame.Camera.performed += OnCameraMove;
+        _controls.Ingame.Jump.performed += OnJump;
+        _characterController = GetComponent<CharacterController>();
+        Cursor.lockState = CursorLockMode.Locked;
+    }
+
+    // Update is called once per frame
```

Comment:
In VR bewegen wir uns ja als "Kamera". Ich versteh noch nicht so ganz wie wir hier den Player steuern. Das hier sieht nach continious movement aus, wollten wir den nicht mit wasd steuern oder versteh ich was falsch? Wenn ich es nur nicht sehe wären ein paar Kommentare gut. Auch für die anderen, wenn wir weiter daran arbeiten.

Reply:
Ich benutze einfach das Unity InputSystem mit einem composite WASD binding. Das Asset mit den Bindings liegt im InputSystem Ordner.